scripts/Release/managedsak_onetime_setup.ps1

```diff
@@ -0,0 +1,89 @@
+<#
+.SYNOPSIS
+    Updates an existing storage account as a keyvault managed storage account.
+.DESCRIPTION
+    Storage account keys need to be autorotated to decrease the risk of having them compromised.
+    Adding a storage account as a keyvault managed storage account lets the keyvault auto rotate
+    the account keys on specified intervals.
+    Since user principal is needed to add a managed storage account, this script needs to be manually run.
+#>
+
+Param(
+    [Parameter(Mandatory=$true)]
+    [String]
+    $subscriptionId,
+
+    [Parameter(Mandatory=$true)]
+    [String]
+    $resourceGroup,
+
+    [Parameter(Mandatory=$true)]
+    [String]
+    $keyVaultName,
+
+    [Parameter(Mandatory=$true)]
+    [String]
+    $storageAccountName,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $upn,
+
+    [Parameter(Mandatory=$false)]
+    [string]
+    $regenerationPeriod = "P60D",
+
+    #The default here is for the msft tenant.
+    #For production/PME, pass in id: e6045dae-1e30-48d4-9502-f77712ec84c8
+    [Parameter(Mandatory=$false)]
+    [string]
+    $keyVaultObjectId = "93c27d83-f79b-4cb2-8dd4-4aa716542e74"
```

Comment:
Is it ok to leave this in open source code?

Reply:
Yes. This is the object id for the cross-tenant service principle that enables this feature.